conans/test/integration/package_command_test.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+from conans.test.tools import TestClient
+from conans.model.ref import ConanFileReference, PackageReference
+import os
+from conans.paths import CONANFILE
+
+
+class PackageCommandTest(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
avoid extra clutter by just defining:
client = TestClient(users=...) inside the package_test()